src/main/utils/makeLinksOpenExternally.ts

```diff
@@ -7,6 +7,7 @@
 import { BrowserWindow, shell } from "electron";
 
 import { Settings } from "../settings";
+import { execSteamURL, isDeckGameMode, openURL } from "./steamOS";
```

Comment:
it'd be good to give this func a more descriptive name
```suggestion
import { execSteamURL, isDeckGameMode, steamOpenURL } from "./steamOS";
```

Reply:
good point yop